src/libcollections/vec.rs

```diff
@@ -1627,6 +1630,7 @@ impl<T> IntoIter<T> {
     #[inline]
     /// Drops all items that have not yet been moved and returns the empty vector.
     #[unstable(feature = "iter_to_vec")]
+    #[deprecated(since = "1.3.0", reason = "replaced by drain()")]
```

Comment:
True, but I believe that this was originally added to get back the original allocation, in which case `drain` suffices for this.